assets/app/scripts/controllers/create/createFromImage.js

```diff
@@ -137,7 +142,14 @@ angular.module("openshiftConsole")
                         message: "Cannot create " + objectName + ". ",
                         details: failure.data.message
                       });
-                      hasErrors = true;
+                    }
+                  );
+                  result.success.forEach(
+                    function(success) {
+                      alerts.push({
+                        type: "success",
+                        message: "Created " + success.kind.toLowerCase() + " \"" + success.metadata.name + "\" successfully. "
```

Comment:
The message is more readable if kind is "deployment config" or even "deployment configuration" vs "deploymentconfig", same with any of the multi-word kinds.  The existing failures are even different, they seem to show it in camel case as "deploymentConfig".   So consistency one way or another would be nice :)